utils.py

```diff
@@ -44,190 +42,19 @@ def to_snake(name):
+        )
 
-    # pos-process
-    cleaned = post_process(changes_md.read_text(), gh_org, gh_repo)
-    changes_md.write_text(cleaned)
-    return changes_md
-
-
-def pre_process(document: str) -> str:
-    replaces = [
-        (
-            "Convert :github: directive",
-            r":github:`(\d+)`",
-            r"`#\1 <https://github.com/%this_org_slash_repo%/issues/\1>`__",
-        ),
-        (
-            "Convert :redmine: directive",
-            r":redmine:`(\d+)`",
-            r"`#\1 <https://pulp.plan.io/issues/\1>`__",
-        ),
-        (
-            "Convert :ref: directive w/ <...>",
-            r":ref:`([a-zA-Z_\s\.]+)\s<[a-zA-Z_\s\.]+>`",
-            r"*\1*",
-        ),
-        (
-            "Convert :meth: directive w/ <...>",
-            r":meth:(`[a-zA-Z_\s\.]+`)",
-            r"\1",
-        ),
-        (
-            "Convert :ref: directive without <...>",
-            r":ref:`([a-zA-Z_\s\.]+)`",
-            r"*\1*",
-        ),
-    ]
-    for _, pattern, repl in replaces:
-        document = re.sub(pattern, repl, document)
-    return document
-
-
-def post_process(document: str, gh_org: str, gh_repo: str) -> str:
-    replaces = [
-        (
-            "Remove # Changelog to include header later",
-            r"#\sChangelog",
-            "",
-        ),
-        (
-            "Replace %this_org_slash_repo% with the plugin name",
-            "%this_org_slash_repo%",
-            f"{gh_org}/{gh_repo}",
-        ),
-        (
-            "Use nice anchor links",  # x.y.z (YYYY-MM-DD) {: #x.y.z }
-            r"## ([0-9]+\.[0-9]+\.[0-9]+) (\([0-9-]+\))",
-            r"## \1 \2 {: #\1 }",
-        ),
-        (
-            r"Remove backslahes before backticks",  # \' \* -> ' *
-            r"\\(['\*`\"\[\]\<\>_~\-\.\$])",
-            r"\1",
-        ),
-        (
-            r"Remove []{.title-ref}",  # \' \* -> ' *
-            r"\[([0-9a-zA-Z\_\s\.\-\/:\(\)\=]+)\]\{\.title-ref\}",
-            r"\1",
-        ),
-        (
-            r"Normalize dash separators",
-            r"\n-+\n",
-            "\n---\n",
-        ),
-        (
-            "Remove double line breaks",
-            r"^\n\n+",
-            "",
-        ),
-    ]
-    for _, pattern, repl in replaces:
-        document = re.sub(pattern, repl, document)
-
-    header = dedent(
-        """\
-        # Changelog
-
-        [//]: # (You should *NOT* be adding new change log entries to this file, this)
-        [//]: # (file is managed by towncrier. You *may* edit previous change logs to)
-        [//]: # (fix problems like typo corrections or such.)
-        [//]: # (To add a new change log entry, please see the contributing docs.)
-        [//]: # (WARNING: Don't drop the towncrier directive!)
-
-        [//]: # (towncrier release notes start)
-
-        """
-    )
-    document = header + document
-    return document
+    return [line.strip()[8:] for line in response.content.decode().split("\n") if "- name:" in line]
```

Comment:
This is a rather crude way to parse yaml...

Reply:
My plan is to do the minimum working solution now and later improve this membership thing (on pulp-docs, also). But I can add a yml library as requirement and make it more robust. Would you prefer that?

Reply:
Will it stay parsing yaml? Or did you plan something completely different?

Reply:
I thought about doing a simple requirement-like file which lists `gh-org/gh-repo` and has `#` comments, but not sure if it's a good idea yet. If not, I might stick with yaml.